python/tskit/trees.py

```diff
@@ -4703,12 +4704,12 @@ def Tajimas_D(self, sample_sets=None, windows=None, mode="site"):
 
         .. code-block:: python
 
-            D = (T - S/h) / sqrt(a*S + (b/c)*S*(S-1))
-            h = 1 + 1/2 + ... + 1/(n-1)
-            g = 1 + 1/2^2 + ... + 1/(n-1)^2
-            a = (n+1)/(3*(n-1)*h) - 1/h^2
-            b = 2*(n^2 + n + 3)/(9*n*(n-1)) - (n+2)/(h*n) + g/h^2
-            c = h^2 + g
+            D = (T - S / h) / sqrt(a * S + (b / c) * S * (S - 1))
+            h = 1 + 1 / 2 + ... + 1 / (n - 1)
+            g = 1 + 1 / 2 ^ 2 + ... + 1 / (n - 1) ^ 2
+            a = (n + 1) / (3 * (n - 1) * h) - 1 / h ^ 2
+            b = 2 * (n ^ 2 + n + 3) / (9 * n * (n - 1)) - (n + 2) / (h * n) + g / h ^ 2
```

Comment:
I'm not so keen on this, much less readable. Could add `nofmt`?

Reply:
Looks like it's not Python (h^2 rather than h**2, etc), so no point in being to zealous about black here.

Reply:
Removed the `python` specification, fixed.